torch/nn/functional.py

```diff
@@ -4847,7 +4847,7 @@ def _in_projection(
      dropout_p (float): Dropout probability; if greater than 0.0, dropout is applied
      need_attn_weights (bool): If true, the second return value will contain the attention weights used;
          otherwise, the second return value is unspecified
-     is_causal (bool): If true, assumes causal attention masking; for this case, attn_mask should not be set.
+     is_causal (bool): If true, assumes causal attention masking and ignores attn_mask.
```

Comment:
Just a heads up I know there is a PR for xformers; https://github.com/facebookresearch/xformers/pull/587  that I think will allow for both casual plus additive attn bias. But I think for now this is the right way to go